### DISCUSSION TITLE
Welcome to fre:ac Discussions!

### DISCUSSION THREAD

**Discussion by enzo1982** [enzo1982/freac#267] [Announcements]
## 👋 Welcome!
The fre:ac project is moving its discussion forums to GitHub. Please post new questions and ideas here and feel free to start discussions about anything releated to fre:ac and digital audio conversion.

The Discussions area is a place to connect with other members of the fre:ac community. We hope that you:

  * Ask questions you’re wondering about.
  * Share your ideas for new features and improvements.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

You can still find the [old discussion forums](https://sourceforge.net/p/bonkenc/discussion/) on SourceForge for reference, but please start new discussions over here.

**Comment by FriskyDog** [comment]
I have been using Fre:AC to split .m4b files into the .mp3 parts. This works quite well for me. Sometimes I have a few very large .mp3 files for a book. I need a way to examine the audio, and then split at silence points with a target length defined at 5 minutes. This target length can vary + or - by several minutes to find a good point for making the split. As this an option for Fre:AC, or would this be better suited for another program?

**Reply by enzo1982** [reply]
fre:ac does not support splitting output at silence yet, but I'm working on an output splitting tool and plan to include such functionality.

So right now you will need to look for a different program to accomplish this, but in the not too far future this could become possible with fre:ac.

**Comment by BlueSky0237** [comment]
I'd like to re-rip my entire CD collection to flac files. Does fre:ac automatically identify, tag, add song titles, album titles and cover art in the process?
